agenda-webapps/src/main/webapp/vue-app/agenda-common/components/event/form/AgendaEventFormBasicInformation.vue

```diff
@@ -58,12 +58,17 @@
               class="text-wrap ml-2" />
           </div>
         </div>
-        <agenda-event-form-conference
-          :event="event"
-          :settings="settings"
-          :current-space="currentSpace"
-          :conference-provider="conferenceProvider"
-          class="my-auto" />
+        <div class="d-flex flex-row">
+          <i class="uiIconVideo darkGreyIcon uiIcon32x32 uiIcon32x32 my-3 mr-11"></i>
+          <div class="d-flex flex-column">
+            <agenda-event-form-conference
+              :event="event"
+              :settings="settings"
+              :current-space="currentSpace"
+              :conference-provider="conferenceProvider"
+              class="my-auto" />
```

Comment:
I have a vertical scrollbar when using a text input for Web conf (when not Web conference provider is configured for Agenda)